.travis.yml

```diff
@@ -65,6 +69,14 @@ script:
   - if [ "$GOOS" = "linux" ]; then sudo PATH=$PATH GOPATH=$GOPATH make integration ; fi
   # Run the integration suite a second time. See discussion in github.com/containerd/containerd/pull/1759
   - if [ "$GOOS" = "linux" ]; then sudo PATH=$PATH GOPATH=$GOPATH TESTFLAGS_PARALLEL=1 make integration ; fi
+  - if [ "$GOOS" = "linux" ]; then
+      sudo PATH=$PATH containerd -log-level debug &> /tmp/containerd-cri.log &
+      sudo PATH=$PATH GOPATH=$GOPATH critest --runtime-endpoint=/var/run/containerd/containerd.sock validation ;
```

Comment:
I'm not sure about this. Why run the entire test suite for cri here?
Isn't this already done upstream?

Reply:
The test only takes 2+min, and I think we agree that we should run some quick presubmit test for CRI integration in containerd?

Reply:
@dnephin this is a small and relatively fast integration test bucket, but we can add skips if there are questionable tests. E.g. we could skip the security tests for certain platforms.